[rospy.client][INFO] 2019-06-25 21:25:55,531: init_node, name[/rostopic_9631_1561497955440], pid[9631]
[xmlrpc][INFO] 2019-06-25 21:25:55,532: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-25 21:25:55,532: Started XML-RPC server [http://dc5b429f784e:38571/]
[rospy.init][INFO] 2019-06-25 21:25:55,533: ROS Slave URI: [http://dc5b429f784e:38571/]
[rospy.impl.masterslave][INFO] 2019-06-25 21:25:55,533: _ready: http://dc5b429f784e:38571/
[xmlrpc][INFO] 2019-06-25 21:25:55,533: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-25 21:25:55,534: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-25 21:25:55,633: registered with master
[rospy.rosout][INFO] 2019-06-25 21:25:55,634: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-25 21:25:55,637: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-25 21:25:55,639: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2019-06-25 21:25:55,747: topic[/fibonacci/feedback] adding connection to [http://dc5b429f784e:33653/], count 0
[rospy.internal][INFO] 2019-06-25 21:25:55,852: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2019-06-25 21:26:45,844: signal_shutdown [signal-2]
[rospy.internal][INFO] 2019-06-25 21:26:45,848: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2019-06-25 21:26:45,848: topic[/fibonacci/feedback] removing connection to http://dc5b429f784e:33653/
[rospy.internal][WARNING] 2019-06-25 21:26:45,849: Unknown error initiating TCP/IP socket to dc5b429f784e:47249 (http://dc5b429f784e:33653/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2019-06-25 21:26:45,851: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 838, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2019-06-25 21:26:45,851: signal-2
[rospy.core][INFO] 2019-06-25 21:26:45,959: signal_shutdown [atexit]
